packages/tezt-bam/tezt-bam.0.2/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+synopsis: "A plugin of [bam] for Tezt"
+description: "Provides a way to register PBT tests with Tezt"
+maintainer: ["François Thiré"]
+authors: ["François Thiré"]
+license: "MIT"
+tags: ["test" "tezt" "pbt" "shrinking" "internal" "bam"]
+homepage: "https://github.com/francoisthire/bam"
+doc: "https://francoisthire.github.io/bam/"
+bug-reports: "https://github.com/francoisthire/bam/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.7" & >= "3.7"}
```

Comment:
```suggestion
  "dune" {>= "3.7"}
```
This is fixed on the repo itself by removing `dune` from the dependency list in the dune-project file (it is added automatically)